apps/api/src/config/env.validators.ts

```diff
@@ -101,4 +101,9 @@ export const envValidators = {
     VERCEL_REDIRECT_URI: url({ default: 'https://dashboard.novu.co/auth/login' }),
     VERCEL_BASE_URL: url({ default: 'https://api.vercel.com' }),
   }),
+
+  CLICK_HOUSE_URL: str({ default: 'http://127.0.0.1:8123' }),
+  CLICK_HOUSE_DATABASE: str({ default: 'default' }),
+  CLICK_HOUSE_USER: str({ default: 'default' }),
+  CLICK_HOUSE_PASSWORD: str({ default: '' }),
```

Comment:
As this serbvice is optiona, maybe we should not add the defaults and allow an empty string and avoid the initializaiton of clickhouse client?